fbuapp/View Controllers/RegisterViewController.h

```diff
@@ -0,0 +1,28 @@
+//
+//  RegisterViewController.h
+//  fbuapp
+//
+//  Created by belchercd on 7/22/19.
+//  Copyright © 2019 eazheng. All rights reserved.
+//
+
+#import <UIKit/UIKit.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@protocol PFUserAuthenticationDelegate;
+
+@interface RegisterViewController : UIViewController
+@property (weak, nonatomic) IBOutlet UITextField *firstNameField;
+@property (weak, nonatomic) IBOutlet UITextField *lastNameField;
+@property (weak, nonatomic) IBOutlet UITextField *usernameField;
+@property (weak, nonatomic) IBOutlet UITextField *passwordField;
+- (IBAction)signupButton:(id)sender;
```

Comment:
Can we move the functions to the bottom next to each other and have all properties grouped together?